lib/i18n/tasks/translators/yandex_translator.rb

```diff
@@ -17,6 +17,8 @@ def initialize(*)
 
     def translate_values(list, **options)
       list.map { |item| translator.translate(item, options) }
```

Comment:
This method should return the result

Reply:
Ah yeah good spot.

These translators are tricky without tests (and much exposure to the code before now)

Reply:
[0ecba9d](https://github.com/glebm/i18n-tasks/pull/606/commits/0ecba9da23fa7c50506c2f3020d257ee14b7ed8a)